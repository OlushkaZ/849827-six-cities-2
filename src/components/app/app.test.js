import React from 'react';
import renderer from 'react-test-renderer';
import App from './app.jsx';

it(`App correctly renders after relaunch`, () => {
  const tree = renderer
 .create(<App
   offers = {[{
     title: ``,
     coast: 0,
     isPremium: true,
     type: ``,
     src: ``,
   }]}
 />)
 .toJSON();
  expect(tree).toMatchSnapshot();
});
