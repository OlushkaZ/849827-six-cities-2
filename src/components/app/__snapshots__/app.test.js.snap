// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after relaunch 1`] = `
<Connect(PageMain)
  offers={
    Array [
      Object {
        "city": "",
        "coast": 0,
        "coordinates": Array [
          0,
          0,
        ],
        "id": "id1",
        "isPremium": true,
        "src": "",
        "title": "",
        "type": "",
      },
    ]
  }
  onClick={[Function]}
/>
`;
